recipe/meta.yaml

```diff
@@ -68,36 +74,37 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}                 # [cuda_compiler_version != "None"]
-    # Dec 2020: it seems that git is broken on windows, so we use m2-git
     - m2-patch  # [win]
-    - m2-git    # [win]
+    - m2-sed    # [win]
     - patch     # [not win]
     - git       # [not win]
```

Comment:
are `patch` and `git` still used at all (beyond stuff that conda-build does internally to apply patches)? Why would we need them in the host environment?

Reply:
Probably not necessary. They used to be required when we cloned the repo I think n

Reply:
Done.